src/renderer/screens/Preferences/UserInterface.js

```diff
@@ -49,6 +50,11 @@ function UserInterfacePreferences(props) {
     setColoredLayoutCards(!coloredLayoutCards);
   };
 
+  const toggleAllowAllKeys = () => {
+    settings.set("ui.allowAllKeys", !allowAllKeys);
```

Comment:
Let's use a setting name that's more descriptive? 

Maybe ui.hideFeaturesNotAvailableInCurrentFirmware? 

It's wordy but also, I think, a lot easier to understand